src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import classNames from 'classnames';
+import { SortCondition } from '../../types/enums';
+
+interface Props {
+  visibleItemsCount: number;
+  filterCondition: SortCondition;
+  isCompletedTodos: boolean;
+  setFilterCOndition: (condition: SortCondition) => void;
+  clearCompleted: () => void;
+}
+
+export const Footer: React.FC<Props> = React.memo(({
+  visibleItemsCount,
+  filterCondition,
+  isCompletedTodos,
+  setFilterCOndition,
+  clearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${visibleItemsCount} items left`}
```

Comment:
Add right condition for `1 item left` todo
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/70009205/2366b773-c06b-4a10-9278-a4ac76795334)